cgmanifest.json

```diff
@@ -8360,19 +8360,19 @@
       "component": {
         "type": "other",
         "other": {
-          "name": "kubernetes",
-          "version": "1.28.3",
-          "downloadUrl": "https://dl.k8s.io/v1.28.3/kubernetes-src.tar.gz"
+          "name": "kube-vip-cloud-provider",
```

Comment:
I think these changes were not intended

Reply:
Nah, they are fine. The script  just put packages in alphabetical order. Looks like that the manifest was just updated manually at some point.:)